geotiler/source/osm.json

```diff
@@ -1,7 +1,6 @@
 {
     "name": "OpenStreetMap",
     "attribution": "© OpenStreetMap contributors\nhttp://www.openstreetmap.org/copyright",
-    "url": "http://{subdomain}.tile.openstreetmap.org/{z}/{x}/{y}.{ext}",
-    "subdomains": ["a", "b", "c"],
+    "url": "http://tile.openstreetmap.org/{z}/{x}/{y}.{ext}",
     "limit": 2
```

Comment:
We need clarification about the tile usage policy for the limit parameter (see https://github.com/wrobell/geotiler/issues/32#issuecomment-1365891196).